molecule/default/playbook_deploy.yml

```diff
@@ -1,13 +1,12 @@
 ---
```

Comment:
this file is not needed delete it and refactor in a single playbook like [this](https://github.com/entercloudsuite/ansible-haproxy/blob/0a2d1f029941216bd17b06e20da9fd7c73b5fa11/packer.json#L30-L31)